3rdPartyCypress/cypress/integration/errorPageNews.js

```diff
@@ -0,0 +1,32 @@
+import {
+  errorMessage,
+  errorPageInlineLink,
+  errorTitle,
+} from '../../../cypress/support/bodyTestHelper';
+import news from '../../../src/app/lib/config/services/news';
+
+// This is a 3rd party test, we should arrange for it to be fixed but it's not due to our application.
```

Comment:
I had to read this comment a couple of times to get it - might be clearer to say "_if it fails_ we should arrange for it to be fixed"?

Reply:
Will fix along with documentation to ensure it's consistent